ios/BT/API/Model/DownloadedPackage.swift

```diff
@@ -14,7 +19,7 @@ struct DownloadedPackage {
       return nil
     }
     do {
-      self = try archive.extractKeyPackage()
+      self = try archive.extractKeyPackage() as! DownloadedPackageImpl
```

Comment:
hm, is there a way we can avoid force-unwrapping here? 🤔

Reply:
done!